libcudacxx/test/libcudacxx/cuda/type_list.pass.cpp

```diff
@@ -719,6 +719,26 @@ static_assert(::cuda::std::is_same<::cuda::std::__type_remove<::cuda::std::__typ
                                    ::cuda::std::__type_list<short, int*>>::value,
               "");
 
+// __type_remove_if
+static_assert(::cuda::std::is_same<::cuda::std::__type_remove_if<::cuda::std::__type_list<>, BiggerThanFour>,
+                                   ::cuda::std::__type_list<>>::value,
+              "");
+static_assert(
+  ::cuda::std::is_same<
+    ::cuda::std::__type_remove_if<::cuda::std::__type_list<char[0], char[8], char[4], char[3], char[5]>, BiggerThanFour>,
+    ::cuda::std::__type_list<char[0], char[4], char[3]>>::value,
+  "");
+
+// __type_copy_if
+static_assert(::cuda::std::is_same<::cuda::std::__type_remove_if<::cuda::std::__type_list<>, BiggerThanFour>,
+                                   ::cuda::std::__type_list<>>::value,
+              "");
+static_assert(
+  ::cuda::std::is_same<
+    ::cuda::std::__type_copy_if<::cuda::std::__type_list<char[0], char[8], char[4], char[3], char[5]>, BiggerThanFour>,
+    ::cuda::std::__type_list<char[8], char[5]>>::value,
+  "");
+
```

Comment:
Empty arrays are invalid in C++
```suggestion
static_assert(
  ::cuda::std::is_same<
    ::cuda::std::__type_remove_if<::cuda::std::__type_list<char[1], char[8], char[4], char[3], char[5]>, BiggerThanFour>,
    ::cuda::std::__type_list<char[1], char[4], char[3]>>::value,
  "");

// __type_copy_if
static_assert(::cuda::std::is_same<::cuda::std::__type_remove_if<::cuda::std::__type_list<>, BiggerThanFour>,
                                   ::cuda::std::__type_list<>>::value,
              "");
static_assert(
  ::cuda::std::is_same<
    ::cuda::std::__type_copy_if<::cuda::std::__type_list<char[1], char[8], char[4], char[3], char[5]>, BiggerThanFour>,
    ::cuda::std::__type_list<char[8], char[5]>>::value,
  "");

```